tests/pyupgrade_test.py

```diff
@@ -47,7 +48,7 @@ def test_roundtrip_text(s):
 )
 def test_intentionally_not_round_trip(s, expected):
     # Our unparse simplifies empty parts, whereas stdlib allows them
-    ret = unparse_parsed_string(parse_format(s))
+    ret = unparse_parsed_string(list(parse_format(s)))
```

Comment:
`list()` shouldn't be necessary here

Reply:
`unparse_parsed_string`'s argument takes a list but i'll try to rework this